stix2validator/output.py

```diff
@@ -149,25 +149,43 @@ def print_results(results):
         results: A list of FileValidationResults instances.
 
     """
-    for file_result in sorted(results, key=operator.attrgetter("filepath")):
-        print_horizontal_rule()
-        print_level(logger.info, "[-] Results for: %s", 0, file_result.filepath)
-
-        if file_result.is_valid:
-            marker = _GREEN + "[+]"
-            verdict = "Valid"
-            log_func = logger.info
-        else:
-            marker = _RED + "[X]"
-            verdict = "Invalid"
-            log_func = logger.error
-        print_level(log_func, "%s STIX JSON: %s", 0, marker, verdict)
-
-        for object_result in file_result.object_results:
-            if object_result.warnings:
-                print_warning_results(object_result, 1)
-            if object_result.errors:
-                print_schema_results(object_result, 1)
-
-        if file_result.fatal:
-            print_fatal_results(file_result.fatal, 1)
+    if not isinstance(results, list):
```

Comment:
We need to update the docstring above. Also, since this function is basically one large `if` statement, should we just split this into two functions, one for `ObjectValidationResults` and one for `FileValidationResults`? I'm slightly concerned about having heterogeneous lists (though I expect this will be relatively rare in practice).